WatchGuard patches to sources for lzo-2.03 as of
Thu May 26 11:36:29 PDT 2022
The patches shown here have been applied to source .tar.gz 
files supplied with the WatchGuard Open Source Archive.

==========================================================================
--- lzo-2.03/autoconf/ltmain.sh.orig	2022-05-26 11:36:26.719500709 -0700
+++ lzo-2.03/autoconf/ltmain.sh	2022-05-26 11:36:26.863494809 -0700
@@ -2761,7 +2761,8 @@
 	      fi
 	    else
 	      # We cannot seem to hardcode it, guess we'll fake it.
-	      add_dir="-L$libdir"
+	      # WG: don't fake it this introduces host pollution
+	      # add_dir="-L$libdir"
 	      # Try looking first in the location we're being installed to.
 	      if test -n "$inst_prefix_dir"; then
 		case $libdir in
